src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonomyType.php

```diff
@@ -32,6 +33,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'form_type' => 'sylius_taxonomy_translation',
                 'label'    => 'sylius.form.taxonomy.name'
             ))
+            ->addEventSubscriber(new BuildTaxonomyFormListener())
```

Comment:
Ok,  I have pushed a new commit doing what I think is necessary for this. Can you clarify the logic behind adding this as a service? I based the initial PR on \Sylius\Bundle\TaxonomyBundle\Form\EventListener\BuildTaxonFormListener which doesn't seem to be defined as a service anywhere that I can see.